presto-bigquery/src/test/resources/test-account.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "presto-bq-credentials-test",
+  "private_key_id": "ffffffffffffffffffffffffffffffffffffffff",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIBVgIBADANBgkqhkiG9w0BAQEFAASCAUAwggE8AgEAAkEAq7BFUpkGp3+LQmlQ\nYx2eqzDV+xeG8kx/sQFV18S5JhzGeIJNA72wSeukEPojtqUyX2J0CciPBh7eqclQ\n2zpAswIDAQABAkAgisq4+zRdrzkwH1ITV1vpytnkO/NiHcnePQiOW0VUybPyHoGM\n/jf75C5xET7ZQpBe5kx5VHsPZj0CBb3b+wSRAiEA2mPWCBytosIU/ODRfq6EiV04\nlt6waE7I2uSPqIC20LcCIQDJQYIHQII+3YaPqyhGgqMexuuuGx+lDKD6/Fu/JwPb\n5QIhAKthiYcYKlL9h8bjDsQhZDUACPasjzdsDEdq8inDyLOFAiEAmCr/tZwA3qeA\nZoBzI10DGPIuoKXBd3nk/eBxPkaxlEECIQCNymjsoI7GldtujVnr1qT+3yedLfHK\nsrDVjIT3LsvTqw==\n-----END PRIVATE KEY-----\n",
```

Comment:
Are we ok to add this in code?

Reply:
this mock crednetial is for testing purpose

Reply:
same concern, let's use obvious fake keys and ids in testing

Reply:
it is a fake keys that looks like a real one that's being used for testing purpose

Reply:
let's use a key not looks like a real one. same for key_id, client_id

Reply:
actually we cannot mock a fake one because the use of google oauth2 for authentication processing on encoding for private key which would result in a failure to validate PKCS8EncodedKey
`java.security.spec.InvalidKeySpecException: java.security.InvalidKeyException: IOException : DerInputStream.getLength(): lengthTag=41, too big.`
This key is a mock private key which has to be legit for google oauth 2.0 authentication